integration_test.py

```diff
@@ -0,0 +1,47 @@
+""" Integration test (i.e. run against a real DC/OS cluster) for dcos-test-utils
+
+Note: this tests expects a DC/OS cluster to already be provisioned.
+
+At a minimum, the following environment variables should be set:
+    DCOS_DNS_ADDRESS
```

Comment:
Why not `DCOS_CLUSTER_URL` or `DCOS_URL`? Similar names are used in [shakedown](https://github.com/dcos/shakedown#usage) and [dcos cli](https://github.com/dcos/dcos-cli#initialization).

Reply:
its historical. If you want to refactor it, that's fine, but I don't think this is the appropriate place

Reply:
If you want to see the history, this is where it came from: https://github.com/dcos/dcos/blob/1.8/packages/dcos-integration-test/extra/conftest.py#L71